### ISSUE TITLE
`FocusLost` event with buffer save makes goneovim window focused back.

### ISSUE THREAD

**Issue by artkpv** [akiyosi/goneovim#306]
In my vimrc: `autocmd FocusLost,BufLeave * silent! wa`

Steps:

1. Open some buffer
2. Modify buffer. Don't save
3. Focus on another window by clicking it for example.

Actual result: the goneovim window focuses back, i.e. from the window I focused it switches back to goneovim window.
Expected result: the second window focused not the goneovim window.  

```
❯ pacman -Q goneovim
goneovim-bin 0.4.13-1
❯ uname -a
Linux  5.15.10-arch1-1 #1 SMP PREEMPT Fri, 17 Dec 2021 11:17:37 +0000 x86_64 GNU/Linux
❯ nvim -v
NVIM v0.6.0
Build type: Release
LuaJIT 2.0.5
Скомпилирован  builduser

Features: +acl +iconv +tui
See ":help feature-compile"

            общесистемный файл vimrc: "$VIM/sysinit.vim"
          значение $VIM по умолчанию: "/usr/share/nvim"

Run :checkhealth for more info
```

**Comment by akiyosi** [comment]
Hi,
I'm trying to reproduce this problem, but it doesn't seem to happen in my archlinux environment.
Is there any possibility of other factors in your environment being involved?

In order to isolate the problem, could you please try the following and see if it reproduces the problem?

```
/path/to/goneovim -u NONE -c "autocmd FocusLost,BufLeave * silent! wa"
```

Also, the latest version of goneovim is v0.5.1 now. Is it possible to check the reproducibility of the problem with the latest version?

**Comment by artkpv** [comment]
Tried it. Run with the command below. Created file, made there a change but didn't save, changed focus. It didn't refocus back. The issue won't reproduce. 

Seems to be because of some plugin?

**Comment by akiyosi** [comment]
It may be some kind of plugin, or a setting you have written in init.vim, etc.

I don't know if the same condition applies to your init.vim, since it seems to have not been updated for about two years.

https://github.com/artkpv/dotfiles/tree/master/.config/nvim

On the other hand, I think this problem is similar to the following issue. The problem reported in this issue has been fixed in the latest 0.5.1, so please check if it is reproduced in the latest version.

https://github.com/akiyosi/goneovim/issues/295

**Comment by artkpv** [comment]
I've updated to 0.5.1 (there was an [issue](https://aur.archlinux.org/packages/goneovim-bin/#news) with the package for Archlinux). The bug won't reproduce now. I used `autocmd FocusLost,BufLeave * silent! wa`, the window focus stays on the same. Thanks!
